src/server/server.js

```diff
@@ -57,43 +84,50 @@ const startServer = async () => {
       path: config.path,
     }),
   );
+  log.info(`[Server] guppy listening on port ${config.port}!`);
+};
 
-  // simple health check endpoint
-  // eslint-disable-next-line no-unused-vars
-  app.get('/_status', statusRouter, (err, req, res, next) => {
-    if (err instanceof CodedError) {
-      // deepcode ignore ServerLeak: no important information exists in error
-      res.status(err.code).send(err.msg);
-    } else {
-      // deepcode ignore ServerLeak: no important information exists in error
-      res.status(500).send(err);
-    }
-  });
+const initializeAndStartServer = async () => {
+  await esInstance.initialize();
+  await startServer();
+};
+// simple health check endpoint
+// eslint-disable-next-line no-unused-vars
+app.get('/_status', statusRouter, (req, res, err, next) => {
+  if (err instanceof CodedError) {
+    // deepcode ignore ServerLeak: no important information exists in error
+    res.status(err.code).send(err.msg);
+  } else {
+    // deepcode ignore ServerLeak: no important information exists in error
+    res.status(500).send(err);
+  }
+});
 
-  // eslint-disable-next-line no-unused-vars
-  app.get('/_version', versionRouter);
 
-  // download endpoint for fetching data directly from es
-  app.post(
-    '/download',
-    downloadRouter,
-    (err, req, res, next) => { // eslint-disable-line no-unused-vars
-      if (err instanceof CodedError) {
-        // deepcode ignore ServerLeak: no important information exists in error
-        res.status(err.code).send(err.msg);
-      } else {
-        // deepcode ignore ServerLeak: no important information exists in error
-        res.status(500).send(err);
-      }
-    },
-  );
+// eslint-disable-next-line no-unused-vars
+app.get('/_version', versionRouter);
 
-  app.listen(config.port, () => {
-    log.info(`[Server] guppy listening on port ${config.port}!`);
-  });
-};
+// download endpoint for fetching data directly from es
+app.post('/download', downloadRouter, (err, req, res, next) => {
```

Comment:
put `// eslint-disable-next-line no-unused-vars` above this to satisfy the linter

Reply:
fixed